src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -479,12 +479,22 @@ namespace NServiceBus.AzureServiceBus
         public System.Threading.Tasks.Task Send(Microsoft.ServiceBus.Messaging.BrokeredMessage message) { }
         public System.Threading.Tasks.Task SendBatch(System.Collections.Generic.IEnumerable<Microsoft.ServiceBus.Messaging.BrokeredMessage> messages) { }
     }
-    public class NamespaceInfo
+    public class NamespaceDefinition : System.IEquatable<NServiceBus.AzureServiceBus.NamespaceDefinition>
```

Comment:
I introduced two new objects: NamespaceDefinition and Namespace**s**Definition.
The first one contains mapping between namespace name and connectionString, the second one contains all the namespace configured in the config phase and it uses the first "value object" as component to store these infos.

Given that, NamespaceDefinition has behaviours to check equality and to setup mapping (name-connectionString). I used the same component into NamespaceInfo to avoid duplicated code. 

I can't use NamespaceInfo during transport configuration phase